.ibm/pipelines/resources/config_map/configmap-app-config-rhdh.yaml

```diff
@@ -64,6 +64,13 @@ data:
           default:
             id: development
             orgUrl: '${GITHUB_ORG_URL}'
+        keycloakOrg:
```

Comment:
Why do you add this one here if it is already in the default dynamic plugin configuration ? Are you trying to override some values ?